### ISSUE TITLE
Strange calculation in --continue file

### ISSUE THREAD

**Issue by mrantonion** [brichard19/BitCrack#307]
It seems that range in --continue file writes incorrectly. Look. My Bitcrack searches files with about 25Mkey/s. It's about 1,5 billion/min. That is 59682F00 in HEX. But 1 minute later in continue.txt file it is written about 37E110000 in HEX that is approx. 10 times larger that must be. After restarting bitcrack it starts searching from that incorrect address (37E110000). Did anyone faced this issue? How it could be fixed?

**Comment by mrantonion** [comment]
P.S. It's all refers to Windows. I tried to compile in Linux (after 15 hours of fixing bugs at source code!!!) ((((( Everything works perfect.

**Comment by sezginyildirim91** [comment]
Is clBitCrack.exe calculating incorrectly?

**Comment by mrantonion** [comment]
> Is clBitCrack.exe calculating incorrectly?

Yes, you are right.

**Comment by mrantonion** [comment]
The issue solved. Just started using cuBitCrack.exe and evrthg run well. :)

**Comment by trextrader** [comment]
> P.S. It's all refers to Windows. I tried to compile in Linux (after 15 hours of fixing bugs at source code!!!) ((((( Everything works perfect.

Can you please let me know how you got around the misaligned address issue for linux?  See my comment under "CuBitcrack not Working on GeForce RTX 3090 Graphics Card #298" open issue...I resolved the first issue in linux for CUDA 11.2 by setting the args for -gencode=arch=compute_75,code="sm_75"  where 75 is equated to the compute capability of my 2080ti = 7.5.  Here is the output as of current running on kali linux 2021.1:

[2021-02-25.00:34:10] [Info] Compression: both
[2021-02-25.00:34:10] [Info] Starting at: 0000000000000000000000000000000000000000000000000000000000000001
[2021-02-25.00:34:10] [Info] Ending at:   FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364140
[2021-02-25.00:34:10] [Info] Counting by: 0000000000000000000000000000000000000000000000000000000000000001
[2021-02-25.00:34:10] [Info] Initializing GeForce RTX 2080 Ti
[2021-02-25.00:34:11] [Info] Generating 262,144 starting points (10.0MB)
[2021-02-25.00:34:11] [Info] 10.0%
[2021-02-25.00:34:11] [Info] 20.0%
[2021-02-25.00:34:11] [Info] 30.0%
[2021-02-25.00:34:11] [Info] 40.0%
[2021-02-25.00:34:11] [Info] 50.0%
[2021-02-25.00:34:11] [Info] 60.0%
[2021-02-25.00:34:11] [Info] 70.0%
[2021-02-25.00:34:11] [Info] 80.0%
[2021-02-25.00:34:11] [Info] 90.0%
[2021-02-25.00:34:11] [Info] 100.0%
[2021-02-25.00:34:11] [Info] Done
[2021-02-25.00:34:11] [Info] Error: misaligned address

Thanks!

/Zap

**Comment by mrantonion** [comment]
It's difficult to say what's the reason of this error in your case because when I tested it on linux I haven't got a 'misaligned address' error... As of CUDA version I used 11.1 toolkit. And I never used the full range of addresses as it meaninglessly.

**Comment by trextrader** [comment]
> It's difficult to say what's the reason of this error in your case because when I tested it on linux I haven't got a 'misaligned address' error... As of CUDA version I used 11.1 toolkit. And I never used the full range of addresses as it meaninglessly.

Yes, that was just to test to see if it was running, speaking of the fuill range, all bitcoin pk's start with 5 uncompressed and L or K I believe if compressed (WIF) I believe.  My background is physics and symbolic logic, not much of a cryptologist but a fast learner. What would be the range then of a standard uncompressed private key which most are either 51 or 52 characters in length represented in hex where for uncompressed the first character is always a 5 followed by the second character which is always a capital letter I think or most of them(?).  I am using CUDA 11.2 and tried rolling kali back to CUDA 10.1 and its not playing very nice...

**Comment by mrantonion** [comment]
You may use converter :)
https://secretscan.org/PrivateKeyWif

**Comment by sari777** [comment]
can please anyone tell me what is this means "56.50Mkeys/s ((40,902,852,608 total)
while in resume file it is has figures in elapsed "915609"
and when it show that end key given range. what does that mean.?
i am little confused.
and i check as test of some known private keys, it just passed away from the given private key but does not show it on the program and found txt file. why is that.?
and can you tell me how can i set the best parameters of -b -t -p on core i7 11th generation.. intel iris xenon grapahics 6gb system.
and how to start cuda in my system.? it is giving this error " Error detecting devices: CUDA driver version is insufficient for CUDA runtime version"
thanks if anyone can help me.
waiting for answer
